lapis/src/main/kotlin/org/genspectrum/lapis/controller/SingleSegmentedSequenceController.kt

```diff
@@ -41,7 +41,8 @@ import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 
 const val IS_SINGLE_SEGMENT_SEQUENCE_EXPRESSION =
```

Comment:
Should we rename this variable to something like
```suggestion
const val SHOW_SINGLE_SEGMENTED_CONTROLLER =
```